src/app/collaborator/collaborator.service.ts

```diff
@@ -18,8 +18,12 @@ export class CollaboratorService {
   getCollaborators(): Observable<User[]> {
     return this.spaces.current.switchMap((space) => {
```

Comment:
This is probably a better solution, check if this works
return this.spaces.current.filter(space => !!space)..switchMap(...)